python_modules/dagster/dagster/core/log_manager.py

```diff
@@ -148,12 +148,51 @@ def construct_log_string(
     )
 
 
+class PythonLogCaptureHandler(logging.Handler):
```

Comment:
Would it make sense to call this `DagsterLogManagerHandler`?  All Python logging handlers receive logs that come from from Python logging loggers.

Reply:
I went with DagsterLogConversionHandler, because DagsterLogManagerHandler kinda makes it sound like it's handling DagsterLogManager logs, which is somewhat the opposite of what it's doing (it's taking logs that were *not* produced by the DagsterLogManager and turning them into Dagster logs).